chapters/en/slides/module3/module3_01.Rmd

```diff
@@ -0,0 +1,274 @@
+and represent the count of observations in each group with bar.
+
+In this plot,
+we can directly see that most movies are around 100 min in length.
+But from our experience,
+we know that factors such as genre and production year,
+can influence movie length.
+
+How can we separate the data based on other columns in the dataset
+and compare multiple distributions?
+
+---
+
+## Comparing multiple distributions via faceting
+
+```{python}
+(alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()')
+ .facet('country'))
+```
+
+Notes:
+Let's start by comparing the movie length broken down per country.
+Here we use faceting which we learnt about in the last module,
+to create one histogram per country.
+
+You can see that this dataset only contains movies made in the UK or the US.
+It does seem like the distribution of movies is pretty equal between the two countries,
+but it is a bit hard to see because there are so many more movies made in the US.
+
+---
+
+## Comparing multiple distributions via faceting with independent y-axes
+
+```{python}
+(alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()')
+ .facet('country')
+ .resolve_scale(y='independent'))
+```
+
+Notes:
+By default when faceting,
+the y-axis is the same for all the subplots.
+This is often the most suitable choice,
+because it gives us an idea of both the shape of the distribution
+and the total number of observations.
+
+If we want to drill down and compare the distribution shape more closely
+regardless of the number of observations
+we could define the y-axes to be independent for each subplot,
+instead of linked to the same axis range.
+
+Now we can see that the distributions might be a bit different,
+it looks like the UK one is slightly bimodal (it has two peaks),
+whereas the US one is unimodal (it has one peak).
+
+We could explore that data further to find out why this is,
+e.g. maybe the genre is more related to movie length
+and the UK movies in this dataset are of different genres than the US ones?
+
+---
+
+## Comparing multiple distributions grouped by two variables
+
+```{python}
+(alt.Chart(movies).mark_bar().encode(
+    alt.X('runtime', bin=alt.Bin(maxbins=30)),
+    y='count()',
+    color='genre')
+ .facet('country')
+ .resolve_scale(y='independent'))
+```
+
+Notes:
+To compare genres and countries in the same plot,
+we need to group on the combinations of these two categorical columns.
+
+We can do this by adding color within one of our faceted subplots,
+so let's color each facet by genre.
+
+We can get a rough indication of the genre differences from this plot,
+but as we have mention previously,
+it is not ideal to be comparing marks with different baselines.
+
+Let's unstack the bars to be able to easier compare the histograms.
+
+---
+
+## Layering bars instead of stacking them can make distributions easier to compare
```

Comment:
I think the slide title does not go well with this viz. I find it very difficult to actually see these distributions with this plot... A density plot would probably be better, as would a ridge plot. I would actually use this plot to motivate those two other options...

Reply:
Isn't this clear from the text where I say the same thing as your comment? I can add "sometimes" to the title and "(density plots)" after "smooth histograms"?

Reply:
I will take out this since it is covered in the previoous modeule